src/Twig/Extension/MediaExtension.php

```diff
@@ -21,12 +21,11 @@
 use Sonata\MediaBundle\Twig\TokenParser\ThumbnailTokenParser;
 use Twig\Environment;
 use Twig\Extension\AbstractExtension;
-use Twig\Extension\InitRuntimeInterface;
 
 /**
  * @final since sonata-project/media-bundle 3.21.0
  */
-class MediaExtension extends AbstractExtension implements InitRuntimeInterface
+class MediaExtension extends AbstractExtension
```

Comment:
BC Break alert

Reply:
Does this matter? I don't think anyone (should) rely on MediaExtension implements InitRuntimeInterface or not.

Reply:
>Does this matter? I don't think anyone (should) rely on MediaExtension implements InitRuntimeInterface or not.

Think about the result if a user is declaring a method (argument or return) with the removed interface. Does this matter?

Reply:
`InitRuntimeInterface` is part of Twig 2 and removed in Twig 3. If the user refers to `InitRuntimeInterface` they can't upgrade to Twig 3.

Reply:
AFAIK, we aren't dropping the support for `twig/twig:2` in this PR.

Reply:
That's true. The functionality of InitRuntimeInterface can be implemented diffently in a way that works both in Twig 2 and 3. InitRuntimeInterface is only used to inject the `Twig\Environment` and I've moved that to the classes in `Twig\Node`.